examples/global-logger/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/cryptopay-dev/yaga/logger/log"
```

Comment:
I would rename to `logger`, not `log` (because this is the name of standard go `log` package)
What do you think?

Reply:
`github.com/cryptopay-dev/yaga/logger/logger`